### PR TITLE
 Bootstrap Assignment 5 & 6

### PR DESCRIPTION
**Pull request by NikitaShivhare1**

### CODE REVIEW

**Review by Pawan1005 — COMMENTED** [review]
where is assignment 6

**Review by shubhamdalal49 — APPROVED** [review]

File: Nikita/Bootstap/Bootstrap Assignment_6/academic.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                        </select>
+                        <simpletext You can select multiple fields.></simpletext>
+                    </div>
+                </fieldset>
+                <br />
+             </form>
+         </div>
+     </div>
+ </body>
+</html>
```
Comment (Pawan1005) [inline Nikita/Bootstap/Bootstrap Assignment_6/academic.html:149]: create proper indentation

File: Nikita/Bootstap/Bootstrap Assignment_6/personal.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    div.main {
+        width: 400px;
+        margin: 100px auto 0px auto;
+    } 
+    h2 {
+        text-align: center;
+        padding: 20px;
+        font-family: sans-serif;
+    }
+</style>
```
Comment (Pawan1005) [inline Nikita/Bootstap/Bootstrap Assignment_6/personal.html:50]: this needs to be in head tag

File: Nikita/Bootstap/Bootstrap Assignment_6/personal.html [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                <input type="radio" name=" gender" id="female">
+                &nbsp;
+                <span id="female"> Female </span>
+                <br><br>
+                <input type="submit" value=" Submit " name="submit" id="submit">
+             </form>
+          </div>
+       </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Nikita/Bootstap/Bootstrap Assignment_6/personal.html:98]: create proper indentation

File: Nikita/Bootstap/Bootstrap Assignment_6/login.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                                Create an Account</p>
+                        </div>
+                    </form>
+              </div>
+            </div>
+         <div class="col-md-4"></div>
+      </div>
+    </div>
+ </body>
+</html>
```
Comment (Pawan1005) [inline Nikita/Bootstap/Bootstrap Assignment_6/login.html]: create proper indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
create proper indentation

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**